crypto/fipsmodule/rand/rand.c

```diff
@@ -94,6 +94,12 @@ struct rand_thread_state {
   // calls is the number of generate calls made on |drbg| since it was last
   // (re)seeded. This is bound by |kReseedInterval|.
   unsigned calls;
+  // last_block_valid is non-zero iff |last_block| contains data from
+  // |get_seed_entropy|.
+  int last_block_valid;
```

Comment:
this field was removed in https://github.com/aws/aws-lc/pull/1112/

Reply:
Removed in latest update.